### PR TITLE
Правки

### PR DESCRIPTION
**Pull request by LaboratorySales**
# PR Details

## Types of changes

- [ ] Dependency change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have updated the documentation accordingly (index.d.ts).

### CODE REVIEW

**Comment by tuyuribr** [comment]
Cool proxy thing but, please keep this in english. And we already have a discussion here: #1949

File: src/Client.js [hunk 1]
```diff
@@ -80,27 +92,66 @@ class Client extends EventEmitter {
...
+                let json = undefined;// ПРОКСИ
+                let ip = undefined;// ПРОКСИ
+                try {// ПРОКСИ
+                    json = JSON.parse(content);// ПРОКСИ
+                    ip = json.ip;// ПРОКСИ
+                    console.log(json);// ПРОКСИ
+                } catch (e) {// ПРОКСИ
+                    console.log(e);// ПРОКСИ
+                }// ПРОКСИ
+                this.emit('proxy_auth', ip);// ПРОКСИ
```
Comment (PurpShell) [inline src/Client.js:145]: very large security risk

**Review by purpshell — CHANGES_REQUESTED** [review]
eeeeek keep it all in english and clean your code please. and also no, we won't blindly accept this pull request. @tuyuribr did you read through the code? Did you see what this person is pushing to mainstream exactly?

**Comment by PurpShell** [comment]
Seeing that a pull request already exists, no thanks.

**Pull request closed without merging**